scripts/globals/items/fishermans_belt.lua

```diff
@@ -0,0 +1,35 @@
+-----------------------------------
+-- ID: 15452
+-- Item: Fisherman's belt
+-- Enchantment: Fishing image support
+-- 2 hours, All Races
+-----------------------------------
+-- Enchantment: Fishing image support
+-- Duration: 2 hours
+-- Fishing Skill +2
+-----------------------------------
+require("scripts/globals/status")
+-----------------------------------
+local item_object = {}
+
+item_object.onItemCheck = function(target)
+    local result = 0
+    if (target:hasStatusEffect(xi.effect.FISHING_IMAGERY) == true) then
```

Comment:
I think just...
"if target:hasStatusEffect(xi.effect.FISHING_IMAGERY) then"
wold suffice

Reply:
Was just mimicing the other belt script verbatim, but yeah, those extra parens..

Reply:
I remember being able to stack different imagery status previously on DSP. This has been fixed by now, surely?

Guess it's time to verify.